cloud/ec2.go

```diff
@@ -198,7 +199,25 @@ func (m *ec2Manager) spawnOnDemandHost(ctx context.Context, h *host.Host, ec2Set
 		if err != nil {
 			return nil, errors.Wrap(err, "problem expanding user data")
 		}
-		userData := base64.StdEncoding.EncodeToString([]byte(expanded))
+		ec2Settings.UserData = expanded
+	}
+
+	if h.Distro.BootstrapMethod == distro.BootstrapMethodUserData {
+		env := evergreen.GetEnvironment()
+		settings := env.Settings()
```

Comment:
the ec2Manager type has a setting object in it, I don't think we need to poll the environment here.